common/models/change.js

```diff
@@ -464,6 +490,9 @@ module.exports = function(Change) {
     }
   };
 
+  Change.handleError = deprecate(Change.handleError,
+    'Change.handleError is deprecated, report your errors via the callback.');
```

Comment:
Well, this message is printed if your code calls `Change.handleError`. I want to tell the user to send the error to the callback of the method which called `handleError`.

```js
Change.custom = function(cb) {
  var self = this;
  someAsyncWork(function(err) {
    // deprecated
    if (err) return handleError(err);
    // the correct way
    if (err) return cb(err);
   
    // do more stuff on success
    cb();
  });
};
```

I don't think anybody is calling `handleError` directly. Of course, I cannot be sure, thus I added this deprecation warning instead of removing the method completely.

How about this message?

```
Change.handleError is deprecated, you should pass errors to your callback instead.
```